### PR TITLE
fix(editableChipInput): fix overflow behavior of editableChipInput component

### LINKED ISSUE #2061: EditableChipInput's characters is going out of field.
description of the bug :

EditableChipInput field have fixed width and after typing something which having more characters ( ex- 30 , 40) then chip field is going out of the input filed width.

1. Go to design system
2. Click on EditableChipInput all button
3. type anything which is having lots of characters and press enter.

Expected behavior :
chip width should be fixed.

Solution :
Fixed the chip width and use hover (when user hover over the chip box then show entire text)

### PR DESCRIPTION
**Pull request by griseduardo**
### What does this implement/fix? Explain your changes.

This PR fix EditableChipInput bug that chip field was going out of the input filled width.

### Does this close any currently open issues?

This PR closes https://github.com/innovaccer/design-system/issues/2061

### Solution

Update maxWidth from `var(--spacing-9)` to `var(--spacing-8)`, by passing maxWidth props in ChipInput

## Before

before-error

## After

after-solution

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Review by anuradha9712 — CHANGES_REQUESTED** [review]
Please submit a pull request with `develop` as the target branch.

**Comment by anuradha9712** [comment]
please create a separate PR for editableChipInput and README file changes.

File: core/components/molecules/chipInput/ChipInput.tsx [hunk 1]
```diff
@@ -225,6 +225,7 @@ export const ChipInput = (props: ChipInputProps) => {
         className="my-3 mx-2"
         onClick={() => onClick && onClick(chip, index)}
         onClose={() => onChipDeleteHandler(index)}
+        maxWidth="var(--spacing-8)"
```
Comment (anuradha9712) [inline core/components/molecules/chipInput/ChipInput.tsx]: There is no need to pass `maxWidth` prop to Chip as it is already accommodated in the `{...rest}` through `chipOptions` prop

**Comment by anuradha9712** [comment]
@griseduardo Thank you for your contribution. Another approach to solve this issue could be to simply pass `maxWidth`  to the `chipInputOptions` prop like the following in the stories file only ([refer to this](https://github.com/innovaccer/design-system/blob/master/core/components/molecules/editableChipInput/__stories__/index.story.jsx))

```
<EditableChipInput 
  chipInputOptions={{
   chipOptions: {{maxWidth: "var(--spacing-8)"}}
  }}
/>
```
Initial maxWidth `var(--spacing-9)` is set to accommodate large number of use-cases. Also there is option available to update the maxWidth based on various use-cases just like in this EditableChipInput case.

Please let me know if this approach works for you or in case you need any more discussion.

**Comment by anuradha9712** [comment]
Please update the commit message to `fix(editableChipInput): fix overflow behavior of editableChipInput component` or something similar to this

**Comment by griseduardo** [comment]
@anuradha9712 Thank you for the suggestions, I applied them here

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**